docs/admin/cluster-large.md

```diff
@@ -75,6 +75,10 @@ However, this is done only for clusters having more than 50 nodes.
 
 ### Addon Resources
 
+** TODO: Update this to reflect
+[https://github.com/kubernetes/kubernetes/pull/16185/](https://github.com/kubernetes/kubernetes/pull/16185/)
+and any other changes since it was written. **
+
 To prevent memory leaks or other resource issues in [cluster addons](../../cluster/addons/) from consuming all the resources available on a node, Kubernetes sets resource limits on addon containers to limit the CPU and Memory resources they can consume (See PR [#10653](http://pr.k8s.io/10653/files) and [#10778](http://pr.k8s.io/10778/files)).
```

Comment:
@dchen1107 Will this still be true after https://github.com/kubernetes/kubernetes/issues/21086?